crates/e2e/tests/e2e/helpers/mod.rs

```diff
@@ -0,0 +1,53 @@
+mod deploy;
+#[macro_use]
+mod onchain_components;
+mod services;
+
+use {
+    anyhow::{anyhow, Result},
+    std::{future::Future, io::Write, time::Duration},
+    tempfile::TempPath,
+};
+pub use {deploy::*, onchain_components::*, services::*};
+
+/// Initialize tracing and assure that panic hook is set.
```

Comment:
Currently doesn't set the panic hook. Is that intended?

Reply:
Thanks, fixed